main.scss

```diff
@@ -5,23 +5,41 @@
 @import 'src/scss/variables';
 @import 'src/scss/mixins';
 
-@if $o-loading-is-silent == false {
+
+/// @param {Map} $opts ['colors': ('light', 'dark'), 'sizes': ('small', 'medium', 'large')
+/// @access public
+// Outputs all loading styles
+@mixin oLoading($opts: (
+	'colors': ('light', 'dark'),
+	'sizes': ('small', 'medium', 'large')
+)) {
+	$colors: map-get($opts, 'colors');
+	$sizes: map-get($opts, 'sizes');
+
+	@include _oLoadingAnimationKeyframes();
 
 	.o-loading {
-		@include oLoading();
+		display: inline-block;
```

Comment:
These three base styles are repeated

Reply:
in `oLoading` and `oLoadingContent` you mean?